# time
float64 time

# boundingbox
int64 center_x
int64 center_y
int64 box_width
int64 box_height
int64 max_length

# camera state
float64 UAV_lat
float64 UAV_lon
float64 UAV_alt
float64 UAV_roll
float64 UAV_pitch
float64 UAV_yaw
float64 cam_roll
float64 cam_pitch
float64 cam_yaw

# laser state
float64 laser_dis

